src/JsonPatch.Tests/packages.config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Bcl.AsyncInterfaces" version="7.0.0" targetFramework="net48" />
+  <package id="MSTest.TestAdapter" version="3.1.1" targetFramework="net48" />
```

Comment:
Had to update the Tests project references as well by removing the below in my screenshot. It is not exactly showing up in the diff
![image](https://github.com/myquay/JsonPatch/assets/93213727/a3f19dd0-d835-4388-abec-a7dce608eee6)